examples/int8_inference_huggingface.py

```diff
@@ -1,27 +1,24 @@
 import torch
-from transformers import AutoModelForCausalLM, AutoTokenizer
+from transformers import LlamaForCausalLM, LlamaTokenizer
 
 MAX_NEW_TOKENS = 128
 model_name = 'decapoda-research/llama-7b-hf'
```

Comment:
```suggestion
model_name = 'meta-llama/Llama-2-7b-hf'
```
The decapoda-research models are deprecated, you should use `meta-llama/Llama-2-7b-hf`